src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/AllPackagesIndexWriter.java

```diff
@@ -89,11 +89,11 @@ protected void buildAllPackagesFile() throws DocFileIOException {
     }
 
     /**
-     * Add all the packages to the content tree.
+     * Add all the packages to the content.
```

Comment:
Should be `adds`

regarding `the content` ... it seems strange to see `the content` used in a way that does not specify _which_ content. That's why I often use `the given content` or `the specified content` in summary sentences. Then, later on, in the context of a `@param` it works to give the shorter form of `the content`